.github/workflows/java-release.yml

```diff
@@ -8,13 +8,196 @@ on:
         description: "The Dafny version to run"
         required: true
         type: string
+      version:
+        description: "The version being released"
+        required: true
+        type: string
+  # Useful to run this action by itself since sometimes to apply hot patches
+  # to just java we need to only release a jva build
+  workflow_dispatch:
+    inputs:
+      dafny:
+        description: "The Dafny version to run"
+        required: true
+        type: string
+      version:
+        description: "The version being released"
+        required: true
+        type: string
 
 jobs:
-  releaseJava:
+  allowList:
+    uses: ./.github/workflows/validate_user.yml
+  releaseToCA:
+    needs: allowList
+    strategy:
+      matrix:
+        library: [AwsCryptographicMaterialProviders]
+        dotnet-version: ["6.0.x"]
+    runs-on: codebuild-AWS-MPL-Release-${{ github.run_id }}-${{ github.run_attempt }}-ubuntu-5.0-large
+    permissions:
+      id-token: write
+      contents: read
+    env:
+      REGION: us-east-1
```

Comment:
Question: If the host is in `us-east-1`, do all our tests know to call `us-west-2` for the test resources?
We may need to override this variable when running tests.

Reply:
yes, all tests know to call us-west-2. This is not new, all CB Release builds are hosted in us-east-1 calling us-west-2 resources